### PR TITLE
Revision: Color Codes

### PR DESCRIPTION
**Pull request by DJStompZone**
Updated `_remove_color_codes` method to strip "§g" ("Minecoin Gold", Bedrock Edition only), added optional `flavor` argument, defaulting to value "java" to match present behavior.

### CODE REVIEW

**Comment by DJStompZone** [comment]
@Sch8ill My bad, I didn't mean to add the Github Actions onto this PR. I can delete them or move to a new PR if you'd prefer to just merge the original changes to mcclient/response.py on this one

**Comment by Sch8ill** [comment]
Hey @DJStompZone ,
thanks for the contribution, i really appreciate it.
The implementation of a regex was long over due, thanks for your help 😁
I think it would be better to move your new worflows into a different PR.
I'm just going to comment a view parts of your code, i hope you are ok with that.

Best regards,
Till

File: mcclient/response.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import datetime
+import datetime, re
```
Comment (Sch8ill) [inline mcclient/response.py]:
```suggestion
import datetime
import re
```
That's the convention i try to follow

**Comment by Sch8ill** [comment]
@DJStompZone Would you mind adding the new Bedrock functionallity to the 'BedrockResponse' class?

File: mcclient/response.py (hunk 1 above)
Comment (DJStompZone) [reply mcclient/response.py]: Got it

**Comment by DJStompZone** [comment]
> Hey @DJStompZone , thanks for the contribution, i really appreciate it. The implementation of a regex was long over due, thanks for your help 😁 

You betcha! 

> I think it would be better to move your new worflows into a different PR. 

So I'm not exceedingly fluent with the more advanced git stuff, but I think I got the workflow stuff moved out of this PR into a seperate branch, so this one should be good to go.

> I'm just going to comment a view parts of your code, i hope you are ok with that.

Go for it

**Comment by DJStompZone** [comment]
> @DJStompZone Would you mind adding the new Bedrock functionallity to the 'BedrockResponse' class?

Sure, can we do that on a new PR though? Just to be completely honest I don't want to confuse myself with all the different commits, git isn't exactly my strongest skill 😅

**Comment by Sch8ill** [comment]
@DJStompZone The PR is really messed up right now because i made some changes to files that you missed to fetch (my bad, i'm sorry). I think the best fix would be  for you to open up a new PR containing only the changes to ```response.py```

**Comment by Sch8ill** [comment]
Don't worry, i'm also not that good at git 😅.
Are you using github desktop?
That really helped me in the beginning

**Comment by DJStompZone** [comment]
> @DJStompZone The PR is really messed up right now because i made some changes to files that you missed to fetch (my bad, i'm sorry). I think the best fix would be for you to open up a new PR containing only the changes to `response.py`

I [rebased and squashed in your changes](https://github.com/Sch8ill/MCClient-lib/pull/15/commits/0a99974949b8162b3d38b513fe03a7e9522c7577), so I *think* it's good to go. It's showing no conflict, but definitely have a look and make sure your latest changes are there.

**Comment by DJStompZone** [comment]
> Are you using github desktop?

Combination of desktop, website, and CLI depending on what I'm doing at the time. GH Desktop is definitely the most intuitive though, for sure

**Comment by Sch8ill** [comment]
> > @DJStompZone The PR is really messed up right now because i made some changes to files that you missed to fetch (my bad, i'm sorry). I think the best fix would be for you to open up a new PR containing only the changes to `response.py`
> 
> I [rebased and squashed in your changes](https://github.com/Sch8ill/MCClient-lib/pull/15/commits/0a99974949b8162b3d38b513fe03a7e9522c7577), so I _think_ it's good to go. It's showing no conflict, but definitely have a look and make sure your latest changes are there.

@DJStompZone
the workflow files are stil in this PR
Could you remove them? (:
Additionally, could you take a look at the changes i suggested for `_remove_color_codes` ?

**Comment by Sch8ill** [comment]
implemented in #17

**Pull request closed without merging**